patches/blockprotocol+0.0.1.patch

```diff
@@ -10,23 +10,47 @@ index 205630a..ccd3499 100644
    pageNumber?: number;
    itemsPerPage?: number;
    multiSort?: BlockProtocolMultiSort | null;
-@@ -116,6 +117,7 @@ export type BlockProtocolUploadFileFunction = {
+@@ -79,6 +80,7 @@ export type BlockProtocolLinkedDataDefinition = {
+ };
+ 
+ export type BlockProtocolAggregateEntitiesPayload = {
++  entityId?: string;
```

Comment:
Sorry I missed this before - `entityId` is not a field on `BlockProtocolAggregateEntitiesPayload`. We don't want to filter an aggregation to a single entity (we could instead just request it)